src/test/java/APISampleTest.java

```diff
@@ -0,0 +1,45 @@
+import com.qaprosoft.apitools.validation.JsonCompareKeywords;
+import com.qaprosoft.carina.core.foundation.IAbstractTest;
+
+import com.qaprosoft.carina.core.foundation.api.http.HttpResponseStatusType;
+import com.qaprosoft.carina.demo.api.fakestoreapi.DeleteMethod;
+import com.qaprosoft.carina.demo.api.fakestoreapi.GetAllMethod;
+import com.qaprosoft.carina.demo.api.fakestoreapi.PostMethod;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
+import org.testng.annotations.Test;
+import java.lang.invoke.MethodHandles;
+public class APISampleTest implements IAbstractTest {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+
+
+    // GET_ALL, Third Test Case (Works)
+    @Test
+    public void getAll() {
```

Comment:
all test method names should have `test` word. rename getAll -> getAllProductsTest